src/Exception/MultipartUploadException.php

```diff
@@ -55,4 +60,26 @@ public function getState()
     {
         return $this->state;
     }
+
+    /**
+     * Get file path of the transfer
+     *
+     * @return array
+     */
+    public function getFilePaths()
```

Comment:
Maybe add some docs to state that the filePath may be null if the bucket and key could not be ascertained from the previous exception. Which, by the way... This class is coupling itself to S3 now even though it's meant to be generically used for both S3 and Glacier.